api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -45,6 +45,11 @@ message UpstreamLocalityStats {
   // aggregated over all endpoints in the locality.
   uint64 total_error_requests = 4;
 
+  // The total number of requests that were issued by this Envoy since
+  // the last report. This information is aggregated over all the
+  // upstream endpoints in the locality.
+  uint64 total_issued_requests = 8;
```

Comment:
Probably warrants an entry in version_history.md.

Reply:
Updated. Please take a look.